AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -543,13 +557,153 @@ async function validateConnectionStringAsync(propertyName, connectionString, typ
     }
     return subChecks;
 }
+async function validateConnectionViaAppSetting(propertyName, connectionString, type, diagProvider, failureDetailsMarkdown = undefined,entityName = undefined) {
+    var checkConnectionStringResult = await diagProvider.checkConnectionViaAppSettingAsync(propertyName, type, entityName).catch(e => {
+        logDebugMessage(e);          
+    });
+
+    var subChecks = [];
+
+    // Suppress successful checks to avoid clutter
+    if (checkConnectionStringResult == undefined) {
+        subChecks.push({
+            title: `Validation of connection string failed due to an internal error. Please send us feedback via the "Feedback" button above."`,
+            level: 2,
+        })
+    } else if (checkConnectionStringResult.StatusText != "Success") {
+        /* Status enums as of Sep 2021
+         *  Success,
+         *  AuthFailure,
+         *  ContentNotFound,
+         *  Forbidden,
+         *  UnknownResponse,
+         *  EndpointNotReachable,
+         *  ConnectionFailure,
+         *  DnsLookupFailed,
+         *  MsiFailure,
+         *  EmptyConnectionString,
+         *  MalformedConnectionString,
+         *  UnknownError
+         */
+        var service, title;
+        var detailsMarkdown = "";
+        switch (type) {
+            case ConnectionStringType.BlobStorageAccount:

```

Comment:
You are not accounting for "StorageAccount" - legacy support for old DaaS

Reply:
fixed